plutus-core/plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -66,6 +66,8 @@ test_applyDefaultBuiltin =
         , test_applyBuiltinFunction TakeByteString
         , test_applyBuiltinFunction DropByteString
         , test_applyBuiltinFunction EqByteString
+        , test_applyBuiltinFunction EncodeUtf8
+        , test_applyBuiltinFunction DecodeUtf8
```

Comment:
Please drop these changes, that module is going to be gone once #3208 is merged.

Reply:
Done 👌